### ISSUE TITLE
Unknown catastrophic failure with Hunt for Giant Rodents fighters guild quest

### ISSUE THREAD

**Issue by zeldafan1593** [Interkarma/daggerfall-unity#1458]
**Describe the bug**
During the Hunt for Giant Rodents quest, the game crashes in a way that (from what I'm assuming) causes my graphics drivers to have to reload, as well as mutes my audio on my PC entirely until I reset the computer.

I say "crash" but the game seems entirely playable after said crash, but with no audio/graphics whatsoever, save for the software mouse when pausing. Saving and presumably loading still functions, so you can quicksave > exit game > load quicksave and continue on with the playthrough.

**To Reproduce**
I'd assume that this can be reproduced by accepting/following through with the quest, though I'll see if I can get a save that showcases the bug. There's a possibility as well that a monster could be spawning inside of me, but I can't exactly tell because the crash is instant.

Edit: I've gotten a save with the bug present, but I've realized I don't know where to find the save files, so that's a bit awkward.

Edit 2: Found out how. Here's the link: https://drive.google.com/open?id=1TPMnRYiLun3bww6vtUEHYZXna9xMWOBe

**Desktop (please complete the following information):**
 - OS: Windows 10 Home 64-bit

**Comment by JayH2971** [comment]
I loaded your saved and completed the quest without issue, so it isn't carrying over there. What are your graphics and audio specs?

**Comment by zeldafan1593** [comment]
https://imgur.com/a/6ouBqN3 There you go

**Comment by ajrb** [comment]
The log file after this issue would be good to have if it's not reproducible from the savefile by Jay.

**Comment by Interkarma** [comment]
I can also play through this quest without issue.

This is the first time something like this has been reported in over 2 years of the quest system being available. It seems unlikely this would be related to Daggerfall Unity or that quest directly without something unique on your side in the mix as well.

There is no facility in quests to crash video drivers or mute sounds. They are very lightweight data processors only and don't have any direct system interaction.

If you are using mods, I recommend removing them and see if the problem persists. Likewise disable any software that might be interacting with the game somehow, like shader injectors, recording software, and custom display drivers. For example, Citrix Workspaces creates a custom display environment that Unity is known to sometimes have issues with.

As Hazelnut said, a copy of your output log would be handy to see if there's anything else we can learn about what's happening. Information on finding saves, logs, and installing the game is below.

https://forums.dfworkshop.net/viewtopic.php?f=5&t=2360

While reading through install steps above, please also confirm that your DFU is extracted into a separate folder to the Daggerfall game files, and that each version of DFU is always unzipped to a clean folder (not on top of the previous version). That has been known to cause some unusual issues in the past, though nothing like what you're experiencing.

If you're able to provide the output log, we might be able to help more. Thank you and good luck!

**Comment by zeldafan1593** [comment]
Well I feel dumb.

I was looking around, and it turns out that my issue was actually on my side. A lightning storm causes a particular outlet to not output as much power as it used to, causing the PC to have insufficient power to run properly. Every time the computer would try to draw a specific amount of power, serious issues would arise.

It just happened that the issue would occur at the exact same place while playing the game. I have since resolved the issue with my PC and confirmed that the save file I produced works properly.

I'm sorry for wasting your time.
